src/app/space/create/deployments/deployments-donut/deployments-donut-chart/deployments-donut-chart.component.html

```diff
@@ -9,13 +9,15 @@
 </div>
 
 <div class="sr-only">
-  <div *ngIf="pods?.total === 0; else elsePods">No pods.</div>
-  <ng-template #elsePods>
-    <div>
-      Pod status:
-      <li *ngFor="let column of pods.pods">
-        <span>{{ column[1] ? column[1] + ' ' + column[0] : '0 ' + column[0] }}</span>
-      </li>
-    </div>
-  </ng-template>
+  <span *ngIf="pods">
```

Comment:
Can this be an ng-container rather than a span?

Reply:
Yes, thanks! Changed.